crowbar_framework/app/views/barclamp/proposal_show.html.haml

```diff
@@ -26,13 +26,14 @@
     = render_barclamp_edit_deployment(@proposal)
 
     .panel-footer.text-right
+      = cancel_barclamp_button(@proposal.barclamp)
       - unless @proposal.pretty_deployment.crowbar_committing
-        = save_proposal_button(@proposal)
         = apply_proposal_button(@proposal)
         = dequeue_proposal_button(@proposal)
         - if @active
           = deactivate_proposal_button(@proposal)
         - else
           = delete_proposal_button(@proposal)
+        = save_proposal_button(@proposal)
```

Comment:
I would keep "Apply" next to "Save" as the two are relatively close in term of intent. So if we move "Save" to the end, maybe move "Apply" just before it?